utils/mariadb/files/mysqld.init

```diff
@@ -21,12 +22,19 @@ unset MY_ARGS MY_GROUP MY_USER
 my_user="${MY_USER:-mariadb}"
 my_group="${MY_GROUP:-mariadb}"
 
+mysql_get_var() {
```

Comment:
I don't like keeping the variable like this. Why not just copy the function from Debian? It looks nice and clean the way they do it.

Reply:
Ok, dropped caching. I think my function is better readable and the main issue is that print-defaults prints only stuff set in configuration file while in my approach you get even default values not set anywhere in configuration file.